core/packages/contracts/src/Types.sol

```diff
@@ -2,18 +2,36 @@
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
 pragma solidity 0.8.20;
 
-type ParaID is uint32;
+type ParaID is uint256;
```

Comment:
Is this type correct? `ParaId` is still a `u32` in the parachain: https://github.com/Snowfork/snowbridge/blob/a99fc50c1ebb23a3fd74e4c28b8bcb08b209b869/parachain/primitives/core/src/lib.rs#L21

Reply:
(resolves to this: https://github.com/paritytech/polkadot/blob/6a71a7ef7/parachain/src/primitives.rs#L154)